package.json

```
@@ -16,37 +16,39 @@
     "build": "rm -rf ./build/dist && tsc -p tsconfig.dist.json",
     "license": "license-check-and-add",
     "publish-guide": "npx styleguidist build && cp -r .circleci styleguide && gh-pages --dist styleguide --dotfiles",
-    "test": "react-scripts-ts test --env=jsdom",
-    "eject": "react-scripts-ts eject"
+    "watch": "react-scripts start",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
   },
   "dependencies": {
     "classnames": "2.2.6",
-    "react": "16.4.2",
-    "react-display-name": "0.2.4",
-    "react-dom": "16.4.2"
+    "react": "16.12.0",
+    "react-display-name": "0.2.5"
   },
   "devDependencies": {
-    "@types/classnames": "2.2.7",
-    "@types/enzyme": "3.1.16",
-    "@types/enzyme-adapter-react-16": "1.0.3",
-    "@types/jest": "23.3.13",
-    "@types/node": "10.12.19",
-    "@types/react": "16.4.1",
-    "@types/react-dom": "16.0.6",
+    "@types/classnames": "2.2.9",
+    "@types/enzyme": "3.10.4",
+    "@types/enzyme-adapter-react-16": "1.0.5",
+    "@types/jest": "25.1.1",
+    "@types/node": "13.7.0",
+    "@types/prop-types": "15.7.3",
+    "@types/react": "16.9.19",
+    "@types/react-dom": "16.9.5",
     "codecov.io": "0.1.6",
-    "enzyme": "3.3.0",
-    "enzyme-adapter-react-16": "1.1.1",
-    "gh-pages": "2.0.1",
-    "license-check-and-add": "2.3.6",
-    "react-docgen-typescript": "1.12.3",
-    "react-scripts-ts": "2.17.0",
-    "react-styleguidist": "7.2.5",
-    "rivet-collapsible": "0.2.10",
-    "rivet-switch": "0.3.1",
-    "rivet-uits": "1.6.0",
-    "semantic-release": "15.9.12",
-    "typescript": "2.9.2",
-    "webpack": "3.8.1"
+    "enzyme": "3.11.0",
+    "enzyme-adapter-react-16": "1.15.2",
+    "gh-pages": "2.2.0",
+    "license-check-and-add": "3.0.4",
+    "react-docgen-typescript": "1.16.2",
+    "react-dom": "16.12.0",
+    "react-scripts": "^3.3.1",
```

This dependency should use the exact version

```suggestion
    "react-scripts": "3.3.1",
```